tests/models/decoder_only/language/test_mistral.py

```diff
@@ -81,3 +127,22 @@ def test_mistral_format(
         name_0="hf",
         name_1="mistral",
     )
+
+
+@pytest.mark.parametrize("dtype", ["bfloat16"])
+@pytest.mark.parametrize("model", MODELS[1:])  # v1 can't do func calling
+def test_mistral_function_calling(
+    vllm_runner,
+    model: str,
+    dtype: str,
+) -> None:
+    with vllm_runner(model,
+                     dtype=dtype,
+                     tokenizer_mode="mistral",
+                     config_format="mistral",
+                     load_format="mistral") as vllm_model:
```

Comment:
We can set `max_model_len` here to reduce the memory usage/

Reply:
I believe this test runs on a single L4 24GB, so I think the model is just a bit too large

Reply:
I'm fine with this. Let's get it merged.